### ISSUE TITLE
Overwrite of existing file possible?

### ISSUE THREAD

**Issue by trollsoft7** [sdaoudi/HomeAssistant-GDrive-Uploader#15]
Hello, first thank you for the useful HA integration. I use it to publish a web-cam-jpg on gdrive for shared access. always same name, so I can share the link. What should I do, that the integration overwrites the existing file (ie. picture.jpg)? Probably a feature request.
Thanks for your efforts!

**Comment by sdaoudi** [comment]
Hello,

I just released version 1.1.1, with this option

A new parameter `override_file` has been added to the service. By default, if the file is already exist in the target directory, it will be not uploaded. You can force the upload by setting override_file to true:

```yaml
script:
  upload_video:
    alias: Upload video
    sequence:
      - action: gdrive_uploader.upload
        data:
          parent_id: 20YTAZESppoiZ4hvuI543diltpei80ij
          upload_file_path: /config/home-assistant.log
          override_file: true
```

**Comment by trollsoft7** [comment]
Thank you, just tried it. It works great! I close the issue. Thx again.
